contracts/BridgingManager.sol

```diff
@@ -13,7 +13,7 @@ contract BridgingManager is AccessControl {
     /// @param isDepositsEnabled Stores the state of the deposits
     /// @param isWithdrawalsEnabled Stores the state of the withdrawals
     struct State {
-        bool isInitialized;
+        bool isInitialized; /// @dev DEPRECATED since bridges have their own code for versioning.
```

Comment:
```suggestion
        bool isInitialized; /// @dev DEPRECATED since v2 as bridges have their own code for initialization and storage versioning.
```